website/client/components/tasks/column.vue

```diff
@@ -19,7 +19,7 @@
     textarea.quick-add(
       :rows="quickAddRows",
       v-if="isUser", :placeholder="quickAddPlaceholder",
-      v-model="quickAddText", @keypress.enter="quickAdd",
+      v-model="quickAddText", @keypress.enter="quickAdd($event); quickAddFocused = false",
```

Comment:
the code looks good but I think the `quickAddFocused = false` should be moved inside the quickAdd function, we try to keep code in templates to the minimum